Formula/fregerepl.rb

```diff
@@ -0,0 +1,29 @@
+class Fregerepl < Formula
+  desc "Formula for installing the Frege REPL (not the compiler!!!)"
```

Comment:
And you can also remove the “Formula for installing the“. That should be a description of the software being installed, not of the formula itself. I'd go as far as suggesting something completely different:

```ruby
desc "REPL (read-eval-print loop) for Frege"
```

I also think the “(not the compiler)” doesn't add any useful information. The description should be about what the software is and not about what it is not.